pages/common/yq.md

```diff
@@ -0,0 +1,36 @@
+# yq
+
+> A lightweight and portable command-line YAML processor.
+> More information: <https://mikefarah.gitbook.io/yq/>.
+
+- Output a YAML file, in pretty-print format (yq v4+):
```

Comment:
Perhaps this will read better?
```suggestion
- Output a YAML file, in pretty-print format (`yq` v4+):
```

Same for the other instances.

Reply:
Or maybe just `v4+`...?

Reply:
@joshuatcasey @sbrl what do you think?

Reply:
I think simpler is better, so how about just "(v4+)".

Reply:
I agree.  Go for it,

Reply:
Yeah, let's go with `(v4+)`.